Correct JES references and other mods to install list (#1285)
keptn/keptn.github.io: content/docs/install/_index.md

   specifying the access method you have chosen:
    * All on one cluster
    * Install using a [Multi-cluster setup](multi-cluster),
    meaning that the Keptn Control Plane is installed in one Kubernetes cluster
    and the Keptn Execution Plane is installed in one or more other Kubernetes clusters.

5. [Authenticate CLI and Bridge](authenticate-cli-bridge)

6. If you have problems with your installation,
   check out the hints in [Troubleshooting the installation](troubleshooting)

At this point, you should have a functional Keptn instance running.

You may also be interested in the following topics:

* [Upgrade Keptn](upgrade)

* [Uninstall Keptn](uninstall)

Additional software may need to be installed and configured
to support the needs of the projects you are running:

* Install software to run Keptn tasks
    * [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
     that runs customizable Keptn tasks as Kubernetes jobs:
        * Install the Integration on the Kubernetes cluster(s)
        where the Keptn Execution Plane is installed.
        * Follow the installation instructions for the integration.

>     * If you are using services that Keptn deploys using `helm-service`,
>       install and configure [Istio](istio) on the Kubernetes cluster(s)
>       where those services are installed.

Reviewer: ```suggestion
    * If you are using `helm-service` to deploy your services through Keptn,
      install and configure [Istio](istio) on the Kubernetes cluster(s)
      where your services are deployed.
```

* If you are using [Quality Gates](../concepts/quality_gates),
   install the [monitoring service](monitoring) you want to use
   following the instructions for that Integration:
   * [DataDog](https://artifacthub.io/packages/keptn/keptn-integrations/datadog-service)
   * [Dynatrace](https://artifacthub.io/packages/keptn/keptn-integrations/dynatrace-service)
   * [Prometheus](https://artifacthub.io/packages/keptn/keptn-integrations/prometheus-service)
   * To use a different Observability Platform as your data source,

>    follow the instructions in [Custom Integrations](../0.18.x/integrations)

Reviewer: This xref needs an update :)

   to create an integration for your Observability Platform.
   or start a conversation in the #keptn-integrations channel in the [Keptn Slack](https://keptn.slack.com).

* Install any Webhook integrations you want

>    and configure [webhooks](webhook_service) them (optional).

Reviewer: This sentence doesn't make sense to me...

5. [Authenticate CLI and Bridge](authenticate-cli-bridge)

6. If you have problems with your installation,
   check out the hints in [Troubleshooting the installation](troubleshooting)

At this point, you should have a functional Keptn instance running.

You may also be interested in the following topics:

* [Upgrade Keptn](upgrade)

* [Uninstall Keptn](uninstall)

Additional software may need to be installed and configured
to support the needs of the projects you are running:

* Install software to run Keptn tasks
    * [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service)
     that runs customizable Keptn tasks as Kubernetes jobs:
        * Install the Integration on the Kubernetes cluster(s)
        where the Keptn Execution Plane is installed.
        * Follow the installation instructions for the integration.
    * Install and configure [Istio](istio) on the Kubernetes cluster(s)

>     where the Keptn Execution Plane is installed.

Reviewer 2: istio is only needed as a requirement of `helm-service`. we should clarify that.

Author: done

Author: done

* If you are using [Quality Gates](../concepts/quality_gates),
   install the [monitoring service](monitoring) you want to use
   following the instructions for that Integration:

>    * DataDog

Reviewer 2: Can we directly link to the github repo of datadog for now, e.g., https://github.com/keptn-sandbox/datadog-service?

Also, should we get someone in charge of creating an entry for datadog-service on our artifacthub repo?

Reviewer 3: Already there: https://artifacthub.io/packages/keptn/keptn-integrations/datadog-service

Author: And text now links to the DataDog page on artifacthub

Author: done.  DataDog is now on artifacthub and the text links to that.

   * [Dynatrace](https://artifacthub.io/packages/keptn/keptn-integrations/dynatrace-service)
   * [Prometheus](https://artifacthub.io/packages/keptn/keptn-integrations/prometheus-service)
   * To use a different Observability Platform as your data source,

>    follow the instructions in [Custom Integrations](http://localhost:1313/docs/0.17.x/integrations/)

Reviewer 2: This links to localhost, might want a relative link

Author: blush!  Is fixed.

Author: fixed
